sample/no-op-release/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo "Be sure to increment version if re-using an environment"
```

Comment:
Is this if the no-op-release is changed? If so that does not look like the usual case and maybe a comment is more appropriate here

Reply:
This script is only relevant when the release changed. I saw "echo" as superior to a comment because it's both in code and shows when the script is run. Anyone running the script should be aware of this condition.